lib/consolidate.js

```diff
@@ -17,6 +17,7 @@
 var fs = require('fs');
 var path = require('path');
 var Promise = require('bluebird');
+var semver = require('semver');
```

Comment:
Please remove this line:
```suggestion
```

Reply:
ah sorry yes that makes sense
changed it